engine/class_modules/sc_death_knight.cpp

```diff
@@ -7810,70 +7812,71 @@ action_t* death_knight_t::create_action( util::string_view name, const std::stri
 // death_knight_t::create_expression ========================================
 
 // Equipped death knight runeforge expressions
-std::unique_ptr<expr_t> death_knight_t::create_runeforge_expression( util::string_view name, bool warning = false )
+std::unique_ptr<expr_t> death_knight_t::create_runeforge_expression( util::string_view expression_str,
+                                                                     bool warning = false )
```

Comment:
The variable was called 'name' for a very good reason : because it is the name of the runeforge.